src/transaction.js

```diff
@@ -19,7 +19,7 @@ Transaction.SIGHASH_NONE = 0x02
 Transaction.SIGHASH_SINGLE = 0x03
 Transaction.SIGHASH_ANYONECANPAY = 0x80
 
-Transaction.fromBuffer = function (buffer, __disableAssert) {
+Transaction.fromBuffer = function (buffer, __disableExcess) {
```

Comment:
`__lenient` or `__noCheckTransactionData`. I find negated names most of the time confusing. Given that you prepended it with __ I wonder if we can get away with flipping it to `__strict` without breaking the API.